### ISSUE TITLE
执行训练模块中提供的finetune demo报错

### ISSUE THREAD

**Issue by dinghuawen** [alibaba/MNN#1017]
MNN版本：Mnn-1.0.0

运行train模块中提供的mobilenetv2 finetune 例子报错，使用语雀文档中提供的模型和数据

root@945200b4b0e2:/user/MNN-1.0.0/MNN-1.0.0/build# ./runTrainDemo.out MobilenetV2Transfer mobilenet_v2_tfpb_train_public.mnn train_dataset/train_images/ train_dataset/train.txt test_dataset/test_images/ test_dataset/test.txt 
train iteration: 0 loss: 1.29526 lr: 1e-05
train iteration: 10 loss: 1.42424 lr: 1e-05
train iteration: 20 loss: 1.33856 lr: 1e-05
train iteration: 30 loss: 1.40949 lr: 1e-05
train iteration: 40 loss: 1.3069 lr: 1e-05
train iteration: 50 loss: 1.32389 lr: 1e-05
train iteration: 60 loss: 1.32044 lr: 1e-05
train, 77, cost time: 73188.796875 ms
test iteration: 10 acc: 60/100 = 60%
test iteration: 20 acc: 140/200 = 70%
test iteration: 30 acc: 260/300 = 86.6667%
test iteration: 40 acc: 800/400 = 200%
test iteration: 50 acc: 1330/500 = 266%
test iteration: 60 acc: 1840/600 = 306.667%
test iteration: 70 acc: 1970/700 = 281.429%
test iteration: 80 acc: 2090/800 = 261.25%
test iteration: 90 acc: 2230/900 = 247.778%
test iteration: 100 acc: 2530/1000 = 253%
test iteration: 110 acc: 2810/1100 = 255.455%
test iteration: 120 acc: 3150/1200 = 262.5%
epoch: 0  accuracy: 2.625
train, 115, cost time: 45971.437500 ms
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Alloc memory error or compute src error in Variable::Input
Segmentation fault (core dumped)

**Comment by yyfcc17** [comment]
这些资源文件是给python demo用的，你python能跑吗？

**Comment by caidongqi** [comment]
请问您这些模型和txt文件是在哪里获取的呢？我在云雀上好像没有找到
